pandas/_libs/testing.pyi

```diff
@@ -0,0 +1,8 @@
+
+
+def assert_dict_equal(a, b, compare_keys: bool = ...): ...
+
+def assert_almost_equal(a, b,
+                        rtol: float = ..., atol: float = ...,
+                        check_dtype: bool = ...,
```

Comment:
I think check_dtype should just be Union[bool, str] right?

Reply:
in the .pyx file its typed as bint, which i think is better anyway; better to make the casting explicit